usaspending_api/etl/management/commands/load_fpds_csv.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+import logging
+import re
+import signal
+import sys
+import os
+from collections import OrderedDict
+
+from django.db import connections, transaction
+from django.core.cache import caches
+import pandas as pd
+
+from usaspending_api.awards.models import TransactionFPDS
+from django.core.management.base import BaseCommand
+
+# This dictionary will hold a map of tas_id -> treasury_account to ensure we don't
+# keep hitting the databroker DB for account data
+TAS_ID_TO_ACCOUNT = {}
+
+# Lists to store for update_awards and update_contract_awards
+AWARD_UPDATE_ID_LIST = []
+
+awards_cache = caches['awards']
+logger = logging.getLogger('console')
+
+
+class Command(BaseCommand):
+    """
+    This command will load a single submission from the DATA Act broker. If
+    we've already loaded the specified broker submisison, this command
+    will remove the existing records before loading them again.
+    """
+    help = "Updates the TransactionFPDS with the correct data from the fpds csv"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--file',
+            dest='fpds_csv',
+            nargs='+',
+            type=str,
+            help='the data broker submission id to load'
+        )
+
+        parser.add_argument(
+            '--fiscal_year',
+            dest="fiscal_year",
+            nargs='+',
+            type=int,
+            help="Year for which to run the historical load"
+        )
+
+        # super(Command, self).add_arguments(parser)
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        fiscal_year = options.get('fiscal_year')
+
+        if fiscal_year:
+            fiscal_year = fiscal_year[0]
+            logger.info('Processing data for Fiscal Year ' + str(fiscal_year))
+        else:
+            fiscal_year = 2017
+
+        def signal_handler(signal, frame):
+            transaction.set_rollback(True)
```

Comment:
Should this be `rollback` instead of `set_rollback`?  I could be wrong, not sure what's going on here.

Reply:
set_rollback is the correct syntax here. the function will kick out, check that this is set, and roll it back.